python/nav/event2.py

```diff
@@ -17,6 +17,8 @@
 Next generation event factory functionality for NAV, based on the Django ORM
 models from nav.models.event.
 """
+from django.utils import six
```

Comment:
You need an `from __future__ import absolute_import` before this, to ensure it still works under Python 2.